processor/elastictraceprocessor/go.mod

```diff
@@ -0,0 +1,65 @@
+module github.com/elastic/opentelemetry-collector-components/processor/elastictraceprocessor
+
+go 1.21.1
+
+toolchain go1.22.4
+
+require (
+	github.com/elastic/opentelemetry-lib v0.7.4-0.20240718095211-7190eff24069
```

Comment:
[**For reviewers**] I have kept the otel-lib pointed to main because the component is still in development. I will fix a version once the component is in a somewhat stable state.